modules/@apostrophecms/area/ui/apos/components/AposWidgetControls.vue

```diff
@@ -69,12 +71,6 @@ export default {
     }
   },
   emits: [ 'remove', 'edit', 'cut', 'copy', 'clone', 'up', 'down', 'update' ],
-  data() {
```

Comment:
This setup wasn't reactive.

Reply:
Indeed, wasn't needed without conditions